File: aula02-managed-csharp/PontoApp/PontoApp.cs
using System;

class PontoApp {

    public static void Main(String[] args)
    {
        Ponto p = new Ponto(5, 7);
        p.print();
        System.Console.WriteLine(
            String.Format("p._x = {0}\n", p._x));
        System.Console.WriteLine(
            String.Format("p._y = {0}\n", p._y));
    }
}

File: aula02-managed-java/PontoApp/PontoApp.java
class PontoApp {

    public static void main(String[] args)
    {
        Ponto p = new Ponto(5, 7);
        p.print();
        System.out.println(
            String.format("p._x = %d\n", p._x));
        System.out.println(
            String.format("p._z = %d\n", p._z));
    }
}